bower.json

```diff
@@ -36,5 +35,13 @@
     "web-component-tester": "^6.0.0",
     "polymer": "^1.9.1",
     "document-register-element": "^1.5.0"
+  },
+  "variants": {
+    "polymer2": {
+      "directory": "bower_components-polymer2",
+      "devDependencies": {
+        "polymer": "^2.0.0"
+      }
+    }
```

Comment:
Why not do the variants identically as Polymer does it? https://github.com/PolymerElements/iron-list/blob/master/bower.json